### ISSUE TITLE
Unable to dump textures from wcw vs nwo games

### ISSUE THREAD

**Issue by ghost** [gonetz/GLideN64#2692]
When i press 'D', nothing happens, while several other games are being able to get dumped.

This happens on all regions and versions (like 1.0, 1.1 and so on) of those games.

Solve this issue when possible.

**Comment by Xii-Nyth** [comment]
yeah 4.0 had a weird set of cercumstances where it decided to work right. when using the new one make sure to check use inaccurate texture coordinates or else high res mip maps wont dump in certain games for certain textures
